package.json

```diff
@@ -51,5 +51,8 @@
     "lint": "standard --fix commands/* helpers/* libs/* index.js",
     "test": "jest",
     "doc": "jsdoc2md --template jsdoc2md/README.hbs libs/*.js helpers/*.js > docs/api.md"
+  },
+  "standard": {
+    "globals": ["jest", "describe", "test", "beforeEach", "expect", "it"]
```

Comment:
I would prefer to use the [eslint-config-cozy-app](https://www.npmjs.com/package/eslint-config-cozy-app) and prettier since we use it in other projects. Definitevely not blocking.